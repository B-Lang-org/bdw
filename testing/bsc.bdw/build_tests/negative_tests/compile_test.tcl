
# Compiles the project
handlers::compile 

# Creates a new project
handlers::new_project project1 

# Compiles the project
handlers::compile 

# Closes the project 
handlers::close_project

# Creates a new project
handlers::open_project project.bspec 

# Sets the compilation type
handlers::set_compilation_type bsc

# Compiles the project
handlers::compile 

# Sets the compilation type
handlers::set_compilation_type make 

# Compiles the current project
handlers::compile

# Closes the project 
handlers::close_project

# Exits the worksptation
exit
