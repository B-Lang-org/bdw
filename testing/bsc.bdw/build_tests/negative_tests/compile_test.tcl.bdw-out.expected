There is no open project
The project top file is not specified.
The project top file is not specified.
The project Makefile is not specified.
